netket/nn/symmetric_linear.py

```diff
@@ -715,6 +714,8 @@ def DenseSymm(symmetries, point_group=None, mode="auto", shape=None, **kwargs):
         sym = HashableArray(np.asarray(symmetries.automorphisms()))
     elif isinstance(symmetries, PermutationGroup) or hasattr(symmetries, "__len__"):
```

Comment:
This line (with the check for `__len__`) looks to me like it should actually be `ininstance(symmetries, Sequence)` or something more descriptive.

Reply:
Agree, check if it is a sequence... `HashableArray` is a `Sequence`, right?

Reply:
`HashableArray` only has a very minimal API, so it's not a `Sequence` I think. I think adding another `if` clause like you did is the best way. (Also because this branch here does `HashableArray(np.asarray(...))` which is unecessary if the input already is one,

I just noticed the `hasattr` here as another small issue independent of your PR.